docs/i18n-languages/polish/CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Wytyczne dotyczące twojego wkładu w FreeCodeCamp
+
+Cześć. 👋
+
+funkcjonowanie freeCodeCamp.org jest możliwe dzięki tysiącom życzliwych wolontariuszy takich jak Ty. Jesteśmy wdzięczni za Twój wkład i cieszymy się, że możemy Cię powitać na pokładzie.
+
+Ściśle egzekwujemy nasz [„Kodeks postępowania”](https://www.freecodecamp.org/code-of-conduct). Poświęć chwilę, aby go przeczytać. Ma tylko 196 słów.
+
+Szczęśliwego rozwijania kodu open source!
+
+## Oto kilka zabawnych sposobów, w które możesz pomóc
+
+Możesz przyczynić się do jednego z tych rzeczy:
+
+ 
+1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```

Comment:
```suggestion
1. [Rozwijaj tę otwartoźródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```
This is the proper translation of "open-source" https://sjp.pwn.pl/poradnia/haslo/otwartozrodlowy;10114.html